js/utils/getBitcoinPrice.js

```diff
@@ -14,11 +14,10 @@ module.exports = function (currency, callback) {
 
     //if this is the first check, show status
     if(!window.btcAverages){
-        showStatus = true;
-        var showStatusTimeout = window.setTimeout(function(){
-            $('#statusBar').removeClass('fadeOut').text(polyglot.t('LoadingBitcoinPrices'));
-        },1000);
-
+        showStatus = app.statusBar.pushMessage({
+            msg: polyglot.t('LoadingBitcoinPrices'),
+            duration: false
```

Comment:
Duration is how long the status message will be shown before it is automatically removed. You can control that length (in milliseconds) or, as in this case, you could disable the auto-removal of that particular message since here we'll remove it on our own. The pushMessage() function returns an object with a remove() method that you can call when you want to remove the message.

Most messages won't need to do that as it will be fine to let them be auto-removed. I believe the default duration is 2500.